### PR TITLE
Add ability for users to ignore certain features

### PR DESCRIPTION
**Pull request by dvargasfuertes**
Hey RoamHacker!

This repo is awesome! It's inspired me to want to start contributing to the Roam ecosystem and I thought what better way to start than by helping you out first! Are you accepting contributions?

The basis of this idea is to allow the user in their Roam DB to be able to configure whichever features they'd like to exclude. They could do so by defining an `ignoredFeatures` array in their `roam/js` block before they load RoamMonkey. A proof of concept is in `index.html`. Let me know what you think!

### CODE REVIEW

**Comment by TfTHacker** [comment]
David,
This is a  very cool idea. I would very much like help with RoamMonkey. This is also very cool and elegant solution to feature control. I am planning on adding soon a settings UI to control these features. Question: would you prefer to have a UI or ability to control from roam/js script?

**Comment by dvargasfuertes** [comment]
Hm what would the UI look like? I think as Roam Monkey grows, being able to limit what scripts get loaded into your Roam DB will become important and am not sure how we do that with UI. I guess there could be a `RoamMonkey` page in your DB that has the settings checked on and off and the script could query that page on initial load.

I also don't see anything preventing us from doing both!

**Comment by kunicki** [comment]
Yes you male very good points. If you are on slack for roamresearch, stop
by #roammonkey channel and we can discuss further. Regarding UI have some
new stuff in the works.

File: _loadMonkey.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
 /* global  loadKeyEvents, toastr, loadTypeAhead, displayHelp, displayStartup, loadJumpToDatePicker */
 
+const ignoredFeatures = typeof window.ignoredFeatures !== 'undefined' ? window.ignoredFeatures : [];
```
Comment (dvargasfuertes) [inline _loadMonkey.js:3]: Needed to attach the variable to the window in order to read it. But was able to confirm on my personal db:
image

Confirmed that in this example that the daily note popup was disabled but other monkey features worked.

As we discussed on slack, I'd be happy to do the next iteration of this by using a Roam page for configuration. Any opinions on page name and format?

**Comment by dvargasfuertes** [comment]
PR Ready for review!

**Review by TfTHacker — APPROVED** [review]

**Comment by TfTHacker** [comment]
@dvargas92495 I brought your code into the rep, and now its on my test glitch site. Will do some testing. Also I renamed the tags in _loadMonkey to be better named.

**Pull request merged**